eng/scripts/Export-API.ps1

```diff
@@ -3,9 +3,25 @@ param (
     [Parameter(Position=0)]
     [ValidateNotNullOrEmpty()]
     [string] $ServiceDirectory,
-    [string] $SDKType = "all"
+    [string] $SDKType = "all", 
+    [switch] $SpellCheckPublicApiSurface
 )
 
+if ($SpellCheckPublicApiSurface -and (Get-Command npx | Measure-Object).Count -eq 0) { 
+    Write-Error "Could not locate npx. Install NodeJS (includes npm and npx) https://nodejs.org/en/download/"
```

Comment:
Tip: instead of `Write-Error` without using other parameters for context, remediation, etc., it's easier to just `throw "message"` and you don't need a separate `exit` like you have here. The end result is the same.